samples/subsys/nvs/src/main.c

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
Samples should have documentation too :(

Reply:
I have updated main.c so to include more documentation. Hope this is OK.